markdown/chapters/appendices/playables.md

```diff
@@ -0,0 +1,278 @@
+and are considered either off-meta, that is to say playable but maybe only
+useful in specific circumstances or with certain metagame portions in mind or
+are perhaps *too cute* to run and should probably be omitted from a list unless
+you really wish to play them (because everyone should be having fun with their
+respective lists!). Each card will have a discussion on how much it has been
+tried, its benefits, what it might replace in a common list and any weaknesses
+or considerations it may have associated with it.
+
+## Quicken
+
+:::row{variant=centered}
+- Quicken
+:::
+
+Quicken is a cantrip effect that can be used to try and help power out your
+Doomsday within an opponent's turn or in other tricky situations for your
+opponent. Unfortunately the benefit of enabling this, even with a replacement
+card draw attached, is not really worth the slot. The deck already boasts a high
+density or card filtering, searching and card draw enablers which means Quicken
+has to compare itself against both cycling effects from cards like :card[SW] and
+cantrips like :card[Pnd], :card[Pre] and :card[Con] all of which are likely the
+cards that would be replaced. If you really wanted to have an effect that allows
+these instant speed interactions (because who doesn't love casting :card[Tsz] in
+an opponent's draw step) then :card[Teferi, Time Raveler] is likely better on
+all accounts other than the colour requirement.
+
+## Tutors
+
+:::row{variant=centered}
+- Grim Tutor
+- Infernal Tutor
+- Profane Tutor
+- Scheming Symmetry
+- Solve the Equation
+- Wishclaw Talisman
+:::
+
+There are many tutor effects available in Legacy however many are considered to
+be very archetype-specific. Some of the options presented have significant flaws
+or limitations that make them unsuitable for current iterations of Doomsday.
+:card[Infernal Tutor] is too reliant on the :card[LED] and works poorly with a
+lot of the interaction slots the deck runs like :card[FoW] and :card[Daze].
+:card[Scheming Symmetry] provides the opportunity for your opponent to either
+find their best piece of interaction or their own enabler to progress their game
+plan, especially dangerous if against other fast combo like Oops All Spells or
+TES. :card[Grim Tutor] and :card[Solve the Equation] both provide solid tutor
+effects but at 3 mana and being sorcery only they are slow. Grim Tutor also has
+a life cost which can be costly to the deck.
+
+Of the options above, :card[Wishclaw Talisman] is probably the most effective in
+terms of usage however it still suffers from not being as fast as the currently
+played alternatives of :card[PT] and :card[LDV] which it would replace and is
+open to opposing permanent hate like :card[Abrupt Decay], :card[Collector Ouphe]
+and :card[Prismatic Ending]. If you were less concerned with speed and more
+concerned about card efficiency and were happy playing a slower game then
+Wishclaw or Solve the Equation would be the most viable alternatives.
```

Comment:
Maybe add a sentence, about wishclaw forcing you to win on the same turn, if you want to fetch doomsday with it